st2common/packaging/debian/postinst

```diff
@@ -3,6 +3,9 @@
 if [ -d /etc/st2 ]; then
   sed -i -r "s~(st2.*)/conf~/etc/\1~g" /etc/st2/st2.conf
   sed -i "s~vagrant~/home/stanley~g" /etc/st2/st2.conf
+
+  # Enable auth by default
+  sed -i "s~enable = False~enable = True~g" /etc/st2/st2.conf
```

Comment:
+1 to explicit configurations for dev, prod.

On a related note: https://github.com/StackStorm/st2/blob/master/st2tests/conf/st2.conf We should rename that.